RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -1009,3 +1009,51 @@ boolean wantToForceDrop(monster enemy)
 	return forceDrop;
 }
 
+boolean canSurviveShootGhost(monster enemy, int shots) {
```

Comment:
This is nice but shouldn't `expected_damage(monster)` return this information rather than requiring scripts to hardcode all this individually?

Reply:
I don't think so.

`expected_damage(monster)` says "if the monster attacks us normally, what damage do we expect to take?". The round we use Shoot Ghost, and all subsequent rounds, the ghost does more damage. But we don't know, from an `expected_damage` call, whether the user intends to use Shoot Ghost or not. Also, Mafia doesn't store the skills used in a particular combat, so it doesn't know whether the user has used Shoot Ghost or not.

Reply:
OK fair enough.